README.md

```diff
@@ -45,6 +45,7 @@ irc = miniirc.IRC(ip, port, nick, channels = None, *, ssl = None, ident = None,
 | `msg(target, *msg)`         | Sends a `PRIVMSG` to `target`.              |
 | `notice(target, *msg)`      | Sends a `NOTICE` to `target`.               |
 | `quote(*msg, force=None)` | Sends a raw message to IRC, use `force=True` to send while disconnected. Do not send multiple commands in one `irc.quote()`, as the newlines will be stripped and it will be sent as one command. |
+| `nick(new_nick)`            | Sends a `/nick` to irc server |
```

Comment:
This list is supposed to be in alphabetical order, and `/nick` should be `NICK`.

`irc.nick` contains the (current) nickname of the IRC client, so this function will be removed from all IRC objects unless it has a different name.